sound/soc/sof/ops.h

```diff
@@ -357,7 +357,7 @@ static inline u64 snd_sof_dsp_read64(struct snd_sof_dev *sdev, u32 bar,
 }
 
 static inline void snd_sof_dsp_update8(struct snd_sof_dev *sdev, u32 bar,
-				       u32 offset, u8 value, u8 mask)
+				       u32 offset, u8 mask, u8 value)
```

Comment:
but that commit only changed the name of the function, the issue is introduced by c28a36b012f1f.
Both patch appears at the same tag in upstream, so better to use the commit which actually is the rootcause?

Reply:
@RanderWang, Can you also update the patch subject and drop the `fixup!` part and rephrase it a bit, so it can be sent upstream asap?

Reply:
@ujfalusi  how about this subject ? Thanks!

Reply:
@RanderWang, I would add the `ops` in the commit subject"
`ASoC: SOF: ops: refine parameters order in function snd_sof_dsp_update8 `

Other than that, it looks great.

Reply:
Thanks, updated